index.html

```diff
@@ -1526,35 +1526,37 @@ <h3 id="lifetime-AudioContext" class="informative">
         </section>
         <section>
           <h3>
-            System resources associated with an AudioContext
+            System resources associated with BaseAudioContext subclasses
           </h3>
           <p>
-            <a>AudioContext</a>s should be considered expensive objects.
-            Creating an <a>AudioContext</a> often involves creating a
-            high-priority thread, and using a low-latency system audio stream,
-            both having an impact on energy consumption. Creating more than one
-            <a>AudioContext</a> in a document is most of the time unnecessary.
```

Comment:
The removal was intentional (the language felt like overstepping) , but perhaps I overdid it. WIll reinstate.